shanghai/2020/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - wikix
```

Comment:
Apologies, but you cannot simply PR yourself into becoming an approver. You must first follow the contributor ladder by becoming an org member, then become a reviewer.